src/config/types.ts

```diff
@@ -128,23 +128,16 @@ export interface ExpectOptsConfig {
     interval: number;
 }
 
-export interface MochaOpts {
-    /** milliseconds to wait before considering a test slow. */
-    slow?: number;
-
-    /** timeout in milliseconds or time string like '1s'. */
-    timeout?: number;
-
-    /** string or regexp to filter tests with. */
-    grep?: string | RegExp;
+export interface MochaOpts extends Omit<Mocha.MochaOptions, "ui"> {
+    ui?: string | ((suite: Mocha.Suite) => void);
```

Comment:
Currently ui field inside mocha specified incorrectly. It says that I can use only: `bdd`, `tdd` and other prepared interfaces. But actually I can specify just a string to the file which will be required by mocha or use `function`